addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -34,7 +34,9 @@
                 <div t-attf-id="collapseDependencies{{ dependency_index }}" class="collapse" aria-expanded="false">
                     <ul>
                         <li t-foreach="dependency_value" t-as="item" t-key="item">
-                            <a t-att-href="item.link" t-out="item.contentToDisplay" target="_blank"/>
+                            <a t-att-href="item.link" target="_blank">
+                                This URL is contained in the '<t t-esc="item.field_name"/>' of the following '<t t-esc="item.model_name"/>': <b t-esc="item.record_name"/>
```

Comment:
Will this be translated in one-go with the t-esc inside like that? (by the way, t-esc is not a thing anymore, it should be t-out :open_mouth:

Reply:
Not but doesn't make it more ideal still, I'll change that, maybe we could do a pass later in our templates to fix other occurences too

```
#. module: website
#. openerp-web
#: code:addons/website/static/src/components/dialog/page_properties.xml:0
#, python-format
msgid "This URL is contained in the '"
msgstr ""

#. module: website
#. openerp-web
#: code:addons/website/static/src/components/dialog/page_properties.xml:0
#, python-format
msgid "' of the following '"
msgstr ""
```

After change, it becomes:
```
#. module: website
#. openerp-web
#: code:addons/website/static/src/components/dialog/page_properties.xml:0
#, python-format
msgid "This URL is contained in the '%s' of the following '%s'"
msgstr ""
```
Good catch